website/docs/installation/weave-gitops-enterprise/airgap.mdx

```diff
@@ -5,7 +5,9 @@ toc_max_heading_level: 4
 pagination_next: "getting-started"
 ---
 
-# Install in Airgap Environments
+import TierLabel from "../../_components/TierLabel";
+
+# Install in Airgap Environments <TierLabel tiers="Enterprise" />
```

Comment:
This whole page is about installing Weave GitOps Enterprise. Not sure if we need this label here.

Reply:
Yeh I went back and forth on this. In the end I decided to be safe in case people got here from a search or link or something and dont see that it is under the EE subsection

Reply:
better safe than sorry.